http_client.rst

```diff
@@ -654,6 +655,26 @@ making a request. Use the ``max_redirects`` setting to configure this behavior
         'max_redirects' => 0,
     ]);
 
+Retry Failed Requests
+~~~~~~~~~~~~~~~~~~~~~
+
+Some times, requests failed because of temporary issue in the server or
+because network issue. You can use the
+:class:`Symfony\\Component\\HttpClient\\RetryableHttpClient`
+client to automatically retry the request when it failed.::
+
+    $client = new RetryableHttpClient(HttpClient::create());
```

Comment:
Could you please add a use statement for the new client? Thanks

Reply:
done